src/generic-container.ts

```diff
@@ -145,6 +147,11 @@ export class GenericContainer implements TestContainer {
       }
     }
 
+    if (this.tarToCopy) {
+      this.tarToCopy.finalize();
+      await container.putArchive(this.tarToCopy, "/");
```

Comment:
Would a user ever want to specify the target path?

Reply:
It is the path at which the TAR archive gets extracted. This means it is also the path against which user provided container paths get resolved. For example if a user invokes `withCopyFileToContainer('file.txt', 'dir/bar.txt')`, then the file will end up in `/dir/bar.txt`. Typically though, it is expected that the user already provides an absolute path (`withCopyFileToContainer('file.txt', '/dir/bar.txt')`), in which case this parameter should have no effect.